at_client/src/main/java/org/atsign/client/util/KeysUtil.java

```diff
@@ -18,68 +18,93 @@
 public class KeysUtil {
     private static final ObjectMapper mapper = new ObjectMapper();
 
-    private static final String rootFolder = System.getProperty("user.dir") + "/keys/";
+    private static String rootFolder = System.getProperty("user.home") + "/.atsign/keys/";
 
-    public  static final String pkamPublicKeyName = "aesPkamPublicKey";
-    public  static final String pkamPrivateKeyName = "aesPkamPrivateKey";
-    public  static final String encryptionPublicKeyName = "aesEncryptPublicKey";
-    public  static final String encryptionPrivateKeyName = "aesEncryptPrivateKey";
-    public  static final String selfEncryptionKeyName = "selfEncryptionKey";
+    public static final String pkamPublicKeyName = "aesPkamPublicKey";
+    public static final String pkamPrivateKeyName = "aesPkamPrivateKey";
+    public static final String encryptionPublicKeyName = "aesEncryptPublicKey";
+    public static final String encryptionPrivateKeyName = "aesEncryptPrivateKey";
+    public static final String selfEncryptionKeyName = "selfEncryptionKey";
 
     public static void saveKeys(AtSign atSign, Map<String, String> keys) throws Exception {
-        _makeRootFolder();
-        File file = getKeysFile(atSign);
-        System.out.println("Saving keys to " + file.getAbsolutePath());
-        if (file.exists()) {
-            //noinspection ResultOfMethodCallIgnored
+        rootFolder += atSign + "_key.atKeys";
+        File file = new File(rootFolder);
+        // if keys do not exist in project ~/, search in dir
+        if (!file.exists()) {
+            rootFolder = System.getProperty("user.dir") + "/keys/" + atSign + "_key.atKeys";
```

Comment:
Suggest make "_key.atKeys" a constant, like
`    public static final String keysFileSuffix = "_key.atKeys";`

Reply:
Good idea! This will certainly prevent typos